app/javascript/.storybook/main.js

```diff
@@ -72,6 +72,7 @@ module.exports = {
         ...config.resolve.alias,
         '@crayons': path.resolve(__dirname, '../crayons'),
         '@utilities': path.resolve(__dirname, '../utilities'),
+        '@images': path.resolve(__dirname, '../../assets/images'),
```

Comment:
A new webpack alias. It's defined here, environment.js, jsconfig.json as well as in the eslint config.